docs/content/overview/what-is-rerun.md

```diff
@@ -19,36 +30,25 @@ Rerun provides an integrated solution for working with multimodal temporal data:
 
 **Flexible ingestion:** Load data from your code via the [SDK](../getting-started/quick-start.md), from storage formats like [MCAP](../howto/mcap.md), or from proprietary log formats. [Extend Rerun](../howto/extend.md) when you need custom types or visualizations.
 
-## What is Rerun for?
-
-Rerun helps you debug, understand, and improve systems that generate rich multimodal data. Here's a concrete example:
-
-### Example: debugging a vacuum cleaning robot
+## Who is Rerun for?
 
-Say you're building a vacuum cleaning robot and it keeps running into walls. A traditional debugger won't help, and text logs aren't enough—the robot may log "Going through doorway" but that won't explain why it thinks a wall is a door.
+Rerun is built for teams developing intelligent physical systems:
 
-What you need is to see the world from the robot's perspective in time:
+- **Robotics engineers** debugging perception, controls, and planning
+- **Perception teams** analyzing sensor data and model outputs
+- **ML engineers** preparing datasets and understanding model behavior
+- **Autonomy teams** developing and testing decision-making systems
 
-* RGB camera feed
-* Depth images
-* Lidar scans
-* Segmentation results (how the robot interprets what it sees)
-* The robot's 3D map of the apartment
-* Detected objects as 3D shapes in the map
-* Confidence scores
-* And more
+If you're working with robots, drones, autonomous vehicles, spatial AI, or any system with sensors that evolve over time, Rerun helps you move faster.
 
-You want to see how all these data streams evolve over time so you can pinpoint exactly what went wrong, when, and why.
+## What Rerun is not
 
-Maybe a sun glare hit a sensor wrong, confusing the segmentation network and leading to bad object detection. Or a bug in the lidar code. Or broken odometry made the robot think it was somewhere else. Rerun helps you find out!
+To set clear expectations:
 
-But visual debugging is just the start. Seeing your data also:
-- Gives you ideas for algorithm improvements
-- Helps identify new test cases to set up
-- Suggests datasets to collect
-- Lets you explain your system to colleagues and stakeholders
-
-And the same data you use for visualization can be [queried](../getting-started/data-out.md) to create clean datasets for training and evaluating your models.
+- **Not a training platform**: Use Rerun with PyTorch, TensorFlow, JAX, etc. We prepare your data; you train your models.
+- **Not a deployment tool**: Rerun helps you develop and understand your systems, not deploy them to production.
+- **Not a robot operating system**: Rerun works with ROS, ROS2, or any other robotics stack.
+- **Not a general BI tool**: We're specialized for physical, multimodal, time-series data.
```

Comment:
I would leave this point out, or use a different term than "BI" that's more known to engineers who are the main target of these docs.